pkgs/tools/misc/findutils/disable-getdtablesize-test.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/tests/test-dup2.c b/tests/test-dup2.c
+--- a/tests/test-dup2.c
++++ b/tests/test-dup2.c
+@@ -157,8 +157,6 @@ main (void)
+       ASSERT (close (255) == 0);
+       ASSERT (close (256) == 0);
+     }
+-  ASSERT (dup2 (fd, bad_fd - 1) == bad_fd - 1);
```

Comment:
Is the value of bad_fd also `getdtablesize()`?

Reply:
Correct